wifiphisher/common/accesspoint.py

```diff
@@ -173,14 +109,8 @@ def start_dhcp_dns(self):
             return False
```

Comment:
Also `mypy` pointed this bug where we were returning a boolean value but it was never checked. What should we do about this?